### ISSUE TITLE
src folder missing?

### ISSUE THREAD

**Issue by usmanbahadur** [MDguna/MyFHIRApp#1]
hello i don't see any src folder  folder of your MyFHIRApp it is not been shared on this branch?

**Comment by amiteshg** [comment]
where is src folder?

**Comment by arjunspear** [comment]
Yes, src folder is missing!

**Comment by dougauerbach** [comment]
Correct, without the 'src' folder, this app will not execute. Is there any plan to fix it? Would be great to see a working demo here!

**Comment by MDguna** [comment]
sorry guys for the trouble. Added the src folder. Let me know if you have any questions.

**Comment by dougauerbach** [comment]
> sorry guys for the trouble. Added the src folder. Let me know if you have any questions.

Thanks!

I am seeing the following error messages when attempting to launch the  FHIR app (for example, using `ng serve --open`)

ERROR in src/main.ts(4,27): error TS2307: Cannot find module './app/app.module'.
src/main.ts(5,29): error TS2307: Cannot find module './environments/environment'.

**Comment by rijvirajib** [comment]
Still missing app.module files
